File: Ejercicio4.hs
module Ejercicio4 where

import System.IO



verificarParImpar :: Int -> String
verificarParImpar n
    | n `mod` 2 == 0 = "Par"
    | otherwise      = "Impar"

main :: IO ()
main = do
    hSetBuffering stdout NoBuffering
    putStr "Ingrese un número entero: "
    numStr <- getLine
    let num = read numStr :: Int
    let resultado = verificarParImpar num
    putStrLn $ "El número es: " ++ resultado

File: Ejercicio4.py


def verificar_par_impar():
    try:
        numero = int(input("Ingresa un número entero: "))
        if numero % 2 == 0:
            print(f"El número {numero} es PAR.")
        else:
            print(f"El número {numero} es IMPAR.")
    except ValueError:
        print("Entrada inválida. Por favor, ingresa un número entero.")

if __name__ == "__main__":
    verificar_par_impar()


